Resources/Prototypes/Magic/forcewall_spells.yml

```
@@ -1,7 +1,7 @@
 - type: entity
   parent: BaseAction
   id: ActionForceWall
-  name: forcewall
+  name: Force Wall #DeltaV
```

```suggestion
  name: Force Wall # DeltaV
```

We always use one space before the # and one after